tda/conftest.py

```diff
@@ -600,6 +600,46 @@ def ios_sim_driver(request, selenium_endpoint, se_prefix):
         sentry_sdk.capture_exception(err)
 
 
+@pytest.fixture
+def ios_dotnet_maui_driver(request, selenium_endpoint, se_prefix):
```

Comment:
could we get Android added? Perhaps both or at least Android if iOS is a problem somehow.

We only have Session Replay for Android right now

Reply:
added but it's not starting in the remote emulator for some reason